xml/ha_clvm.xml

```diff
@@ -209,18 +206,18 @@ cLVM) for more information and details to integrate here - really helpful-->
 <screen>&prompt.root;<command>crm</command> configure
 &prompt.crm.conf;<command>primitive</command> clvmd ocf::heartbeat:clvm \
```

Comment:
This is already corrected in the `develop` branch, commit e6004d6. I won't correct it here to avoid merge conflicts. 😉